webapp/src/views/Console/DomainList/NewDomainDialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
```

Comment:
This file seems to be unrelated to the record set management?

Reply:
dead flie, removed